client/src/components/DayAvailability/useStyles.ts

```diff
@@ -0,0 +1,42 @@
+import { makeStyles } from '@material-ui/core/styles';
+import { relative } from 'path';
+
+const useStyles = makeStyles((theme) => ({
+  root: {
```

Comment:
Suggestion: You can use a `<Box>` component above and most of the CSS properties can be passed as props to the Box component directly.

Eg: `<Box display="flex" alignItems="center"......>`